sdk/v1/data_test.go

```diff
@@ -1,4 +1,4 @@
-package sdk_test
+package v1
```

Comment:
I think that this was done on purpose. See: https://medium.com/@benbjohnson/structuring-tests-in-go-46ddee7a25c#52f8